packages/plugin-v2/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 0.5.2 (2021-04-25)
```

Comment:
1. do it in separate commit
2. should be 0.6.2 see https://github.com/kaltura/playkit-js-hotspots/releases/tag/v0.6.1